<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>831</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>470</y>
       <width>251</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_join">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>470</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>join</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="guard_pw">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="guard_id">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>150</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>291</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="let_id">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>230</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>360</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="let_pw">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>290</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="let_nick">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>350</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>470</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_create">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>470</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>가입</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>371</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>로그인 성공!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/java.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>370</y>
       <width>221</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/IC.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="go_zero">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>430</y>
       <width>191</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>처음으로</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLineEdit" name="let_add">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="let_insert">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>350</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>761</width>
       <height>311</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_insertRow">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>410</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>470</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_insert">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>410</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_printM">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>470</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Print_Multi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_print">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Print</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>470</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>421</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_PTE">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Print Text Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_SF">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>250</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Set Font</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_CTE">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>200</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Text Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_SFI">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>300</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Font Italic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_SFCR">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>350</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Font Color Red</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Spin_FS">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>410</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>410</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Font Size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
